### ISSUE TITLE
How to add a threshold in softmax scores

### ISSUE THREAD

**Issue by mfxss** [tensorflow/tensorflow#19973]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
1.8
- **Python version**: 
3.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
When doing multi-classcification usually I got a softmax score and predictoins with below,
```
softmax_scores = tf.nn.softmax(logits=self.scores, dim=-1)
prediction=tf.argmax(self.scores, 1, name="predictions")
```
If the softmax_socres I got is [0.5,0.2,0.3].The prediction is [0]. Now I want to add thresholds 0.6 to softmax_socres.Which means the prediction expected here is [4] which means others.
The only way that I can figure out is to convert softmax_scores to numpy and handle it. Which is out of the model defination.

### Source code / logs
This is only a demo, the full code is too long.
```
import tensorflow as tf
import numpy as np
a=tf.constant(np.arange(6),shape=(3,2))
b=tf.reduce_max(a,1)
c=tf.to_int32(a>3)
d=tf.argmax(c,1)
with tf.Session() as sess:
    sess.run(tf.global_variables_initializer())
    print(c.eval(),d.eval())
```
Here the result I expect is [4,4,2]

**Comment by asimshankar** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
